api/capi/src/nnstreamer-capi-tizen.c

```diff
@@ -723,9 +724,9 @@ ml_tizen_mm_convert_element (ml_pipeline_h pipe, gchar ** result,
     }
 
     /* read ini, type CONFIGURE_TYPE_MAIN */
-    err =
-        _mmcamcorder_conf_get_info (0, (char *) MMFW_CONFIG_MAIN_FILE,
-        &cam_conf);
+    ini_name = g_strdup (MMFW_CONFIG_MAIN_FILE);
+    err = _mmcamcorder_conf_get_info (0, ini_name, &cam_conf);
+    g_free (ini_name);
```

Comment:
```_mmcamcorder_conf_get_info``` accepts ```const char *```. How this fixes the issue?

Reply:
Ah.,... fix Line 94. You seem to have incorrect function signature there!
```
_mmcamcorder_conf_get_info (int type, char *ConfFile, camera_conf ** configure_info);
```
==>
```
_mmcamcorder_conf_get_info(MMHandleType handle, int type, const char *ConfFile, camera_conf **configure_info);
```

Reply:
Or we may include libmm's devel pkg and header if it's "Tizen".